### ISSUE TITLE
bitnami/cert-manager cannot get resource "leases" in API group "coordination.k8s.io"

### ISSUE THREAD

**Issue by Dryymoon** [bitnami/charts#6930]
**Which chart**:
bitnami/cert-manager
version: cert-manager-0.1.4

**Describe the bug**
any certificate not issued and cert-manager-controller print this error:
```
leaderelection.go:325] error retrieving resource lock kube-system/cert-manager-controller: leases.coordination.k8s.io "cert-manager-controller" is forbidden: User "system:serviceaccount:cert-manager:cert-manager-controller" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "kube-system"
```

**To Reproduce**
1. Install cert-manager
```
helm repo add bitnami https://charts.bitnami.com/bitnami
helm repo update
helm install \
  cert-manager bitnami/cert-manager \
  --namespace cert-manager \
  --create-namespace \
  --set installCRDs=true
```
2. Install any service with ingress which configured to request certificates via cert manager
3. Certificates not issued.

**Expected behavior**
Certificates issued.

**Version of Helm and Kubernetes**:

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-16T11:56:40Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"w
indows/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.8", GitCommit:"5575935422cc1cf5169dfc8847cb587aa47bac5a", GitTreeState:"clean", BuildDate:"2021-06-16T12:53:07Z", GoVersion:"go1.15.13", Compiler:"gc", Platform:"
linux/amd64"}

 helm version
version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}
```

**Extra**
Maybe this will help
https://github.com/open-telemetry/opentelemetry-operator/issues/149
https://github.com/open-telemetry/opentelemetry-operator/pull/151/files

**Comment by alemorcuq** [comment]
Thanks for reporting this, @Dryymoon. Have you tried if adding that rule to the role fixes the issue for you?

**Comment by Dryymoon** [comment]
I`m not tried to add rule.
I tried another helm chart "jetstack/cert-manager" which has necessary changes and works fine.

**Comment by alemorcuq** [comment]
Are you interested on contributing with a PR, @Dryymoon ?

**Comment by alemorcuq** [comment]
Fixed by #7035
